opensrp-wcaro/src/main/assets/json.form/observation_illness.json

```diff
@@ -87,11 +87,13 @@
         "hint": "Action taken",
         "values": [
           "Managed",
-          "Referred"
+          "Referred",
+          "None"
```

Comment:
changed exist in pending PR #230 
as per issue #210 text should be "No action taken"
also need to change in JsonFormUtils line 161(check the pending PR) for map